packages/squiggle-lang/src/rescript/ReducerInterface/ReducerInterface_ExpressionValue.res

```diff
@@ -0,0 +1,60 @@
+/*
+  Irreducible values. Reducer does not know about those. Only used for external calls
+  This is a configuration to to make external calls of those types
+*/
+module AE = Reducer_Extra_Array
+module ErrorValue = Reducer_ErrorValue
+
+type rec expressionValue =
+  | EvBool(bool)
+  | EvNumber(float)
+  | EvString(string)
+  | EvSymbol(string)
+  | EvArray(array<expressionValue>)
+  | EvRecord(Js.Dict.t<expressionValue>)
+
+type functionCall = (string, array<expressionValue>)
+
+let rec show = aValue =>
+  switch aValue {
+  | EvBool(aBool) => Js.String.make(aBool)
```

Comment:
Why use Js.String.make here, instead of some bool_to_string method?

Reply:
RESOLVED